changedetectionio/store.py

```diff
@@ -445,6 +445,15 @@ def save_datastore(self):
 
         while True:
             if self.stop_thread:
+                # Suppressing "Logging error in Loguru Handler #0" during CICD.
+                # Not a meaningful difference for a real use-case just for CICD.
+                # the side effect is a "Shutting down datastore thread" message
```

Comment:
![image](https://github.com/dgtlmoon/changedetection.io/assets/61624808/6ff5b387-29eb-450e-ac97-73bf263a1c92)

Reply:
This is just a little side effect.